src/dprint.rs

```diff
@@ -4,21 +4,19 @@
 //! This is unsafe! It is asynchronous with normal UART1 usage and
 //! interrupts are not disabled.
 
-use esp32c3_hal::{pac::UART1, prelude::nb};
```

Comment:
We can remove this module completely now, and use esp_println with the log crate instead. (If we use the log crate we can optimize logs out later by setting a feature)

Reply:
Let's address this it another PR.